CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+## Prerequisites
```

Comment:
Oh looks like this section is empty now?

Reply:
added info from demo md